packages/browser-utils/src/metrics/resourceTiming.ts

```diff
@@ -55,6 +57,22 @@ export function resourceTimingToSpanAttributes(resourceTiming: PerformanceResour
     // For TTFB we actually want the relative time from timeOrigin to responseStart
     // This way, TTFB always measures the "first page load" experience.
     // see: https://web.dev/articles/ttfb#measure-resource-requests
-    'http.request.time_to_first_byte': (resourceTiming.responseStart ?? 0) / 1000,
-  };
+    'http.request.time_to_first_byte':
+      resourceTiming.responseStart != null ? resourceTiming.responseStart / 1000 : undefined,
+  });
+}
+
+/**
+ * Remove properties with `undefined` as value from an object.
+ * In contrast to `dropUndefinedKeys` in core this funciton only works on first-level
+ * key-value objects and does not recursively go into object properties or arrays.
+ */
+function dropUndefinedKeysFromObject<T extends object>(attrs: T): T {
```

Comment:
this is probably useful to be extracted into browser or core at some point. I know we had our reservations against `dropUndefinedKeysFromObject` but IIRC this was mainly due to over-usage and it recursively going through the object. I think a light-weight implementation like this one should be fine but happy to hear other opinions.